src/c++/perf_analyzer/report_writer.cc

```diff
@@ -95,7 +96,13 @@ ReportWriter::GenerateReport()
         ofs << "Avg latency,";
       }
       ofs << "request/response,";
-      ofs << "response wait";
+      ofs << "response wait,";
+      if (should_output_metrics_) {
```

Comment:
What happens if GPU metrics can't be gathered? The data will still exist but be empty?

Reply:
If the `Metrics` object passed in has empty maps, the csv will include the commas for those metric columns but no data will be between the commas. This seems like fine behavior.